.github/workflows/deploy-pnpm.yaml

```diff
@@ -0,0 +1,48 @@
+# Workflow intended to properly build the GitHub Pages Site. Using PNPM 
+
+name: Deployment Docs - PNPM 
+
+on:
+  # Trigger deployment on every push to main branch.
+  push:
+    branches: [main]
+  # Trigger deployment manually.
+  workflow_dispatch:
+    
+jobs:
+  deploy:
+    runs-on: ubuntu-latest 
+    
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # Fetch all commits to get last updated time or other git log info 
+          fetch-depth: 0
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '14'
+      - name: Setup PNPM 
+        run: npm install -g pnpm@v7.0.0
```

Comment:
Is there a specific reason to use a strict version over something like `pnpm@7` (or the latest minor version `pnpm@7.12`?)

Reply:
Closing since its a duplicate, but we will apply the above conversation on both this comment, and the previous one.